w3af/plugins/crawl/open_api.py

```diff
@@ -458,4 +469,9 @@ def get_long_desc(self):
         During parsing an Open API specification, the plugin looks for parameters
         which are passed to endpoints via HTTP headers, and enables them for further testing.
         This behavior may be disabled by setting 'discover_fuzzable_headers' configuration parameter to False.
+
+        By default, URLs discovered by this plugin allow for injection at path parameter
```

Comment:
Not sure if the description is needed here, since IMHO if this feature is implemented it will become the default and people that want to a) understand how openapi fuzzing works, b) disable this feature can go to the parameter documentation.

Reply:
OK, removed this change and improved (hopefully!) the docs at the parameter level.